README.md

```diff
@@ -13,3 +13,32 @@ Please open an issue if you have any questions.
 ### Small Logo
 
 [![Logo Small](gotify-logo-small.png)](gotify-logo-small.png)
+
+### Notification Logo
+
+Logo for the notification drawer was based on the original logo and drawn by
+Choi Wong and has been optimized by Pander Musubi. All rights are owned by the
```

Comment:
May I ask who is Choi Wong? is that someone you know?

Reply:
Yes, a designer I know.